docs-client/src/lib/transports/grpc-unframed.ts

```diff
@@ -41,7 +41,11 @@ export default class GrpcUnframedTransport extends Transport {
     const endpoint = this.getDebugMimeTypeEndpoint(method, endpointPath);
 
     const hdrs = new Headers();
-    hdrs.set('content-type', GRPC_UNFRAMED_MIME_TYPE);
+    // protocol=gRPC is added to mime types for the purpose of endpoint detection
```

Comment:
For context, there is this comment

https://github.com/line/armeria/blob/main/grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/GrpcDocServicePlugin.java#L219

In some sense, this is just the reverse side of that same comment inside the client

Reply:
Thanks for adding the comment. Makes sense.